pkg/compactor/block_visit_marker.go

```diff
@@ -18,24 +18,26 @@ import (
 	"github.com/thanos-io/thanos/pkg/block/metadata"
 )
 
-const BlockVisitMarkerFile = "block.visit"
+// BlockVisitMarkerFile is the known json filename for representing the most recent compactor visit.
+const BlockVisitMarkerFile = "visit-mark.json"
 
 var (
 	ErrorBlockVisitMarkerNotFound  = errors.New("block visit marker not found")
 	ErrorUnmarshalBlockVisitMarker = errors.New("unmarshal block visit marker JSON")
 )
 
 type BlockVisitMarker struct {
-	CompactorID string    `json:"compactorID"`
-	VisitTime   time.Time `json:"visitTime"`
+	CompactorID string `json:"compactorID"`
+	// VisitTime is a unix timestamp of when the block was visited (mark updated).
+	VisitTime int64 `json:"visitTime"`
```

Comment:
I am thinking about adding another `version` field here for possible changes in the future and backwards compatibility.

Reply:
Yeah.. i like that!

Reply:
Will we introduce version in this PR or separate one?

Reply:
I can add it to this pr, let me update